src/components/pages/DrawingSub/RenderDrawingSub.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Header } from '../../common';
+import { Row } from 'antd';
+import { UploadDocs } from '../../common/';
+import { postNewDrawingSub } from '../../../api/index';
+import { SubmissionModal } from '../../common/index';
+
+const RenderDrawingSub = () => {
+  const inst =
+    'Once you finish your drawing, please take a picture of all of your pages and upload them. After all pages are uploaded, click submit.';
+  const submissionId = 1;
+
+  return (
+    <>
+      <Header title="READY, SET ...DRAW!" />
+      <SubmissionModal instructions={inst} />
+      <div className="writing-sub-container">
+        <Row className="main-row">
+          <p>Draw a picture of your favorite part of the story.</p>
```

Comment:
Eventually, this might need to be more dynamic since the prompts change from week to week